frontends/web/src/routes/account/summary/summarybalance.tsx

```diff
@@ -53,6 +58,23 @@ export function SummaryBalance ({
     }, {} as TAccountCoinMap);
   };
 
+  const onSyncAddressesCount = useCallback((
+    code: string,
+    syncedAddressesCount: number,
+  ) => {
+    setSyncStatus((prevSyncStatus) => ({
+      ...prevSyncStatus,
+      [code]: syncedAddressesCount,
+    }));
+  }, []);
+
+  useEffect(() => {
+    unsubscribe(unsubscribeList.current);
+    accounts.forEach(account => {
+      unsubscribeList.current.push(syncAddressesCount(account.code, onSyncAddressesCount));
+    });
+  }, [accounts, onSyncAddressesCount]);
```

Comment:
If possible, I think it would make sense to have the subscription in BalanceRow component, maybe then you could use `useSubscribe` and remove the forEach?

also the type SyncStatus should probably not be in accountsummary.tsx

Reply:
Very good point! Thanks for the catch :) Actually, moving the logic to `BalanceRow` we don't need `SyncStatus` at all. I moved it, but I didn't use `useSubscribe` since we'd also need to modify `syncAddressesCount` which is also used in `account.tsx`. I think it is better to change that in the new PR I opened. wdyt?